use_sim_time: true
graph: grid_5_5
init_bots: 1
init_locations: ''
done: false
sim_length: 20000
random_string: test
algo_name: random_mrpp